test/test_sanity.py

```diff
@@ -752,7 +752,7 @@ def test_embuilder_wildcards(self):
   def test_embuilder_with_use_port_syntax(self):
     restore_and_set_up()
     self.run_process([EMBUILDER, 'build', 'sdl2_image:formats=png,jpg', '--force'])
-    self.assertExists(os.path.join(config.CACHE, 'sysroot', 'lib', 'wasm32-emscripten', 'libSDL2_image_jpg-png.a'))
+    self.assertExists(os.path.join(config.CACHE, 'sysroot', 'lib', 'wasm32-emscripten', 'libSDL2_image-jpg-png.a'))
```

Comment:
Is the `_ => -` change needed, or for aesthetics? If it isn't needed then maybe it isn't worth it, as users might have hardcoded CI commands about it.

Reply:
Tend to use `-` for all our variants (not `_`) so this is just for consistency.

The precise names of these variants should not be hardcoded anywhere.. if it were I think that would be a bug.

Reply:
I just worry about a user that has
```
cp libSDL2_image_jpg-png.a $OUT_DIR
```
in their CI workflow. This change would make them need to rename.

But if you are confident that is unlikely, and we haven't had issues with such changes in the past, sgtm as is.